exasol/ai/mcp/server/mcp_server.py

```diff
@@ -379,6 +409,8 @@ def describe_script(
     def execute_query(
         self, query: Annotated[str, Field(description="select query")]
     ) -> ExaDbResult:
+        if not self.config.enable_read_query:
+            raise RuntimeError("Query execution is disabled.")
```

Comment:
Unrelated to the ticket, and probably not need, but added anyway for completeness, as all other tools have this check.

Reply:
Better safe than sorry. However, I think the tool is actually not added, if I remember correctly.

Reply:
Yes, the tool is not added, so it's like an extra precaution.